templates/user/edit.html.twig

```diff
@@ -3,6 +3,8 @@
 {% block title %}Modification profil{% endblock %}
 
 {% block body %}
+<body class="safeNightGray">
```

Comment:
base.html.twig already has an opening body, doesn'it?

in base.html.twig:

    <body class="{% block bodyClass %}{% endblock %}">

in user/edit.html.twig:

    {% block bodyClass %}safeNightGray{% endblock %}

    {% block body %}
    <h1 class="pageTitle">Modifier mon profil</h1>
    ...

Reply:
oh thank you very much, I understand it more clearly now !